libs/domain/content/src/models/content.model.ts

```diff
@@ -1,10 +1,13 @@
 import { ContentEntities } from '../services';
 
 export interface ContentQualifier {
-  article?: string;
+  type?: string;
+  id?: string;
   entities?: ContentEntities;
 }
 
 export interface Content {
-  article?: string;
+  heading: string;
+  description: string;
```

Comment:
should we introduce a `meta` object here, and store description in there?

Reply:
not sure, IMO plain object simplier for understanding. But I can introduce if we need it